src/main/java/adris/altoclef/tasks/slot/MoveInaccessibleItemToInventoryTask.java

```diff
@@ -40,6 +40,12 @@ protected Task onTick(AltoClef mod) {
             if (_target.matches(StorageHelper.getItemStackInCursorSlot().getItem())) {
                 slotToMove = Optional.of(CursorSlot.SLOT);
             }
+    // issue is a full cursor slot when trying to clear out bad items.
```

Comment:
This resolves the bug where the bot endlessly switches between items in an attempt to clear the 2x2 grid. We handled clearing the grid elsewhere, but this needs to happen also.